homeassistant/components/sky_hub/device_tracker.py

```diff
@@ -1,63 +1,71 @@
 """Support for Sky Hub."""
 import logging
-import re
 
-import requests
+from pyskyqhub.skyq_hub import SkyQHub
 import voluptuous as vol
 
 from homeassistant.components.device_tracker import (
     DOMAIN,
     PLATFORM_SCHEMA,
     DeviceScanner,
 )
-from homeassistant.const import CONF_HOST, HTTP_OK
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
 import homeassistant.helpers.config_validation as cv
 
 _LOGGER = logging.getLogger(__name__)
-_MAC_REGEX = re.compile(r"(([0-9A-Fa-f]{1,2}\:){5}[0-9A-Fa-f]{1,2})")
+_ERROR = logging.ERROR
+_INFO = logging.INFO
```

Comment:
The error and info constants aren't used.

Reply:
Strange the listing didn't spot that, but vulture has